src/components/AverageWinLoss/AverageWinLoss.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent, Fragment } from 'react'
+import { withTranslation } from 'react-i18next'
+import {
+  Button,
+  Card,
+  Elevation, Intent, Position, Tooltip,
+} from '@blueprintjs/core'
+import { DateInput, TimePrecision } from '@blueprintjs/datetime'
+import _sortBy from 'lodash/sortBy'
+
+import Loading from 'ui/Loading'
+import NoData from 'ui/NoData'
+import LineChart from 'ui/LineChart'
+import TimeframeSelector from 'ui/TimeframeSelector/TimeframeSelector'
+import RefreshButton from 'ui/RefreshButton'
+import { isValidTimeStamp } from 'state/query/utils'
+import {
+  momentFormatter, DEFAULT_DATETIME_FORMAT,
+} from 'state/utils'
+import { platform } from 'var/config'
+
+import { propTypes, defaultProps } from './AverageWinLoss.props'
+import parseChartData from './AverageWinLoss.helpers'
+
+const CURRENCIES = ['USD', 'JPY', 'EUR', 'GBP']
+
+class AverageWinLoss extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.handleStartDateChange = this.handleDateChange.bind(this, 'start')
```

Comment:
any specific reason that you don't use arrow function to auto binds those fn ?

Reply:
that's the original style of the project, normally i use arrow functions

Reply:
I think let's just use arrow functions then.

Reply:
@jarindr changed to arrow functions